examples/apps/colorapp/ecs/colorgateway-base-task-def.json

```diff
@@ -0,0 +1,112 @@
+{
+  "family": $NAME,
+  "proxyConfiguration":{
```

Comment:
will it be simpler to transclude this and envoy container below to avoid repetition?

Reply:
Moved envoy container configuration to a different file.

Reply:
just curious, why not move proxyConfiguration too?

Reply:
I think you may have a valid reason, because it relies on app-port. We can do it as a follow up or wait for CloudFormation support.